robot.py

```diff
@@ -0,0 +1,111 @@
+import asyncio
+from contextlib import asynccontextmanager
+
+from viam.components.board import Board
+from viam.components.servo import Servo
+from viam.robot.client import RobotClient
+from viam.rpc.dial import DialOptions
+
+
+@asynccontextmanager
+async def create_robot(creds, address):
+    """
+    This makes a connection to the hardware, creates a Robot object, and then
+    closes the connection when the context manager exits.
+    """
+    opts = RobotClient.Options(
+        refresh_interval=0,
+        dial_options=DialOptions(credentials=creds)
+    )
+    client = await RobotClient.at_address(address, opts)
+    robot = Robot(client)
+
+    await robot.start()
+    try:
+        yield robot
+    finally:
+        await robot.stop()
+        await client.close()
+
+
+class Robot:
+    UPPER_POSITION = 30
+    LOWER_POSITION = 0
+    WIGGLE_AMOUNT = 5
+    INACTIVITY_PERIOD_S = 5
+
+    def __init__(self, client):
+        """
+        This class is in charge of raising and lowering the robot's hand, and
+        wiggling the hand if it has been raised for too long.
+
+        WARNING: this class is not thread safe!
+
+        The client passed in is a RobotClient object.
+        """
+        self._client = client
+        self._servo = Servo.from_robot(self._client, "servo")
+
+        # This will become an asyncio.Task when the hand is raised. It will
+        # wiggle the hand when it has been raised for over INACTIVITY_PERIOD_S
+        # seconds.
+        self._wiggler = None
+
+    async def start(self):
+        """
+        Ideally, this would happen in __init__, but it needs to be async.
+        """
+        await self._servo.move(self.LOWER_POSITION)
+
+    async def stop(self):
```

Comment:
I'm not wild about the names `start` and `stop`, but couldn't think of better ones. Any ideas?